[2024-10-01T16:42:26.780+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: third_task_test.compare_and_trigger manual__2024-10-01T16:34:22.130372+00:00 [queued]>
[2024-10-01T16:42:27.008+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: third_task_test.compare_and_trigger manual__2024-10-01T16:34:22.130372+00:00 [queued]>
[2024-10-01T16:42:27.010+0000] {taskinstance.py:1359} INFO - Starting attempt 3 of 4
[2024-10-01T16:42:27.048+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): compare_and_trigger> on 2024-10-01 16:34:22.130372+00:00
[2024-10-01T16:42:27.078+0000] {standard_task_runner.py:57} INFO - Started process 86 to run task
[2024-10-01T16:42:27.089+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'third_task_test', 'compare_and_trigger', 'manual__2024-10-01T16:34:22.130372+00:00', '--job-id', '95', '--raw', '--subdir', 'DAGS_FOLDER/third_task.py', '--cfg-path', '/tmp/tmpa0e3n6tu']
[2024-10-01T16:42:27.113+0000] {standard_task_runner.py:85} INFO - Job 95: Subtask compare_and_trigger
[2024-10-01T16:42:27.285+0000] {task_command.py:415} INFO - Running <TaskInstance: third_task_test.compare_and_trigger manual__2024-10-01T16:34:22.130372+00:00 [running]> on host f6937622df26
[2024-10-01T16:42:27.505+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='third_task_test' AIRFLOW_CTX_TASK_ID='compare_and_trigger' AIRFLOW_CTX_EXECUTION_DATE='2024-10-01T16:34:22.130372+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-01T16:34:22.130372+00:00'
[2024-10-01T16:42:27.510+0000] {logging_mixin.py:151} INFO - Attempting to read from bucket: huber-chatbot-project, key: sitemap_data/sitemap_data_2023.json
[2024-10-01T16:42:27.534+0000] {base.py:73} INFO - Using connection ID 'aws_default' for task execution.
[2024-10-01T16:42:27.536+0000] {connection_wrapper.py:340} INFO - AWS Connection (conn_id='aws_default', conn_type='aws') credentials retrieved from login and password.
[2024-10-01T16:42:29.731+0000] {logging_mixin.py:151} INFO - All versions:
[2024-10-01T16:42:29.732+0000] {logging_mixin.py:151} INFO - There are not enough versions available.
[2024-10-01T16:42:29.755+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-10-01T16:42:29.774+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=third_task_test, task_id=compare_and_trigger, execution_date=20241001T163422, start_date=20241001T164226, end_date=20241001T164229
[2024-10-01T16:42:29.852+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-10-01T16:42:29.883+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
